src/components/test/calculate.test.js

```diff
@@ -1,6 +1,7 @@
 import calculate from '../logic/calculate';
```

Comment:
- [ ] Great job for testing the logic in the `calculate` function. Now let's equally test all the logic in the `operate` function still in the logic folder.
- [ ] You have only added snapshots test for the Calculator component. As requested by the previous reviewer, kindly add snapshots tests for Quotes and Home components. This will be seen in the __snapshots__ folder. Right now we have only the Calculator snapshot in the folder.